packages/data/src/navigation/selectors.js

```diff
@@ -16,3 +16,7 @@ export const getFavorites = ( state ) => {
 export const isNavigationRequesting = ( state, selector ) => {
 	return state.requesting[ selector ] || false;
 };
+
+export const getPersistedQueryFromStore = ( state ) => {
```

Comment:
```suggestion
export const getPersistedQuery = ( state ) => {
```

`FromStore` seems a little redundant.

Reply:
This is actually what I originally had, but then I ended up with two separate `getPersistedQuery()` functions within the `NavigationPlugin` component (the other imported from Nav utils), which seemed confusing. So I made this one more specific.